buildspec_publish_to_sar.yml

```diff
@@ -6,17 +6,21 @@ phases:
       java: openjdk11
     commands:
       - pip3 install aws-sam-cli
+      - apt install -q -y jq
 
   build:
     commands:
       # Use AWS SAM to build and package the application by using AWS CloudFormation
       - sam build
-      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged.yaml --force-upload
+      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged_raw.yaml
   post_build:
     commands:
+      # Getting latest version tag from git
+      - APPLICATION_VERSION=$(curl -s  https://api.github.com/repos/BIBSYSDEV/$GIT_REPO/releases/latest | jq -r '.tag_name')
+      # Updating metadata.labels in template
+      - envsubst '${CODEBUILD_RESOLVED_SOURCE_VERSION},${GIT_REPO}' < sampackaged_raw.yaml > sampackaged.yaml
       # publishing to SAR
-      - sam publish --template sampackaged.yaml
-
+      - sam publish  --semantic-version $APPLICATION_VERSION  --template sampackaged.yaml
```

Comment:
Maybe add a comment that we are setting SemanticVersion here and not in Metadata section.

Reply:
Will do!